pkg/ntpmonitor/config.go

```diff
@@ -16,12 +16,18 @@
 package ntpmonitor
 
 import (
+	// a blank import is recommended by the Go docs
+	// when using embed with byte slices
+	_ "embed"
 	"fmt"
 	"os"
 
 	"gopkg.in/yaml.v3"
 )
 
+//go:embed ntpsync.yaml
```

Comment:
After reading the embed docs, I believe the file we want to embed needs to be in the package directory calling the embed directive or a sub directory so the file was moved to this directory.

Reply:
Nice! This looks great